HeterogeneousCore/SonicTriton/test/unittest.sh

```diff
@@ -42,22 +42,11 @@ else
        exit 0
 fi
 
-if type singularity >& /dev/null; then
-	echo "has singularity"
-else
-	echo "missing singularity"
+if [[ $(sh singularity-check.sh >/dev/null; echo $?) -ne 0 ]]; then
```

Comment:
couldn't this just be (& similarly for the other script below):
```bash
if ! singularity-check.sh; then
```
(Omitting ` > /dev/null` because I would prefer not to suppress the more detailed printouts that have been factored out into `singularity-check.sh`, as these are useful for debugging failures.)

Reply:
Yes, changing it now. Thanks!